app/components/conversations/conversation-part-closed.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  classNames: ['conversation-part-closed'],
+
+  closedAt: null,
+
+  currentUser: service(),
+
+  isSelf: computed('author', 'user', function() {
+    return get(this, 'author.id') === get(this, 'user.id');
+
+    closedUser: computed ('isSelf', 'currentUser', function() {
```

Comment:
This is the component I'm talking about. Hoping that this will compute what author will be passed into the HBS.

Reply:
Your reasoning here is good, but remember, the component will spell out `You closed this` or `{{username}} closed this`, so you don't really care about the `closedUser`. Your `isSelf` is correct, so what you need now is to modify your template and replace `{{#if currentUser}}` with `{{#if isSelf}}`.